cluster-provision/gocli/opts/rookceph/manifests/pool-test.yaml

```diff
@@ -9,5 +9,12 @@ metadata:
   name: replicapool
   namespace: rook-ceph # namespace:cluster
 spec:
+  failureDomain: osd
```

Comment:
@mhenriks not completely sure about this but it looks like more testing appropriate according to examples